templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -731,11 +731,24 @@ Resources:
         ConfigSetName: !Ref AWS::StackName
         NodeVolumeSize: !Ref NodeVolumeSize
         KubernetesVersion: !FindInMap [ Versions, current, EKS ]
+#shared resources for multi cluster
```

Comment:
*nit*": that comment doesn't really make sense, we want the shared resources be (auto-)created regardless if we run multi or single cluster, no?